fontir/src/serde.rs

```diff
@@ -87,15 +85,25 @@ impl From<StaticMetadata> for StaticMetadataSerdeRepr {
             glyph_locations,
             names: from.names,
             misc: from.misc.into(),
-            glyph_order: from
-                .glyph_order
-                .into_iter()
-                .map(|n| n.as_str().to_string())
-                .collect(),
         }
     }
 }
 
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct GlyphOrderSerdeRepr(Vec<String>);
```

Comment:
does this need to be public?

Reply:
The entire module is pub(crate) in lib.rs so I thought I didn't need to care. However, I have subsequently come to appreciate that pub(crate) here would let the compiler check for unused items so I think I should indeed go set things here pub(crate).

This PR already has a lot of noise so I'm inclined to tackle separately; deferred to https://github.com/googlefonts/fontc/issues/362.